ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>In this tutorial, you perform distributed ML inference via image classification using Apache Spark.</p><p> </p><p>(remove the rest)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>